### PR TITLE
feature: ecommerce landing page

### PR DESCRIPTION
**Pull request by sunilvr**
added home page for ecommerce website

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,203 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap');
+@import url('https://elegantui.netlify.app/styles.css');
+
+html{
+    font-size: 100%;
```
Comment (MayankKumar10) [inline styles.css]: font size by default 100% you don't have to define it here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    <link href="styles.css" rel="stylesheet" />
+    <link href="products.css" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" />
+    <script src="https://kit.fontawesome.com/5085ec57f6.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="wrapper-landing">
+        <section class="section-header">
```
Comment (sainath7878) [inline index.html]: Correct semantic would be header I guess.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+:root{
+    --color-primary:#991b1b;
+    --color-secondary:antiquewhite;
+}
+
+.wrapper-landing{
+    display:flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    height: 100%;
```
Comment (MayankKumar10) [inline styles.css]: The height as well by default is 100%.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+                <button class="primary-btn-icon">
+                    <span class="badge-icon">
+                        <span class="badge badge-green p-sm">4</span>
+                        <i class="fas fa-cart-plus" aria-hidden="true"></i>
+                    </span> 
+                    <span class="txt-bold p-lg">Cart</span>
+                </button>
+            </div>
+        </section>
+        <section class="section-main">
```
Comment (sainath7878) [inline index.html]: Semantic would be main

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+}
+
+.section-header{
+    width:100%;
+    min-height: 6rem;
+    max-height:10%;
+    background-color: var(--color-secondary);
+    display: flex;
```
Comment (MayankKumar10) [inline styles.css:23]: create a helper class for flex so you can also use this elsewhere

**Comment by sunilvr** [comment]
adding images here for reference (since responsiveness has not been given a detailed look yet)

image


image


image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+                                    <p class="txt-bold p-lg">Nutrition, Supplements & more</h2>
+                                    <p class="card-price p-lg">Up to 40% off</p>
+                                    <p class="card-price p-sm">Top Selling!</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+            </div>
+        </section>
+        <section class="section-footer">
```
Comment (sainath7878) [inline index.html]: footer would be correct semantic

File: products.css [hunk 7]
```diff
@@ -0,0 +1,13 @@
+.card-prod-mini {
+  max-width: 15rem;
+  min-height:15rem;
```
Comment (sainath7878) [inline products.css]: give a space after :, it increase readability

**Comment by sunilvr** [comment]
site address :   https://sunil-ecommerce.netlify.app/

**Review by sainath7878 — COMMENTED** [review]
Rest looks fine. I would suggest you to add a screenshot, so that reviewer gets a look of the UI. Code looks clean. Great work.

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+    border-radius: 5px;
+}
+  
+.section-header button,
+.primary-btn-link{
+    background-color: var(--color-secondary);
+    color:var(--color-primary);
+    padding:0.1rem 0.1rem;
+    margin:0.2rem 0.2rem;
+    width:auto;
```
Comment (MayankKumar10) [inline styles.css]: Width by default is auto, and If the value is the same for the top,bottom and left,right you can use one value,

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+}
+
+.search-bar button {
+    font-size: 1.25rem;
```
Comment (MayankKumar10) [inline styles.css:71]: create a helper class for font size

**Review by MayankKumar10 — APPROVED** [review]
Look into these Everything else looks good

**Review by sainath7878 — APPROVED** [review]

File: styles.css (hunk 8 above)
Comment (sunilvr) [reply styles.css]: changes done

File: styles.css (hunk 3 above)
Comment (sunilvr) [reply styles.css]: changes done

File: styles.css (hunk 1 above)
Comment (sunilvr) [reply styles.css]: changes done

**Pull request merged**